angular.json

```diff
@@ -17,7 +17,7 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "outputPath": "dist/ngx-sticky-demo",
+            "outputPath": "dist/browser",
```

Comment:
It's a automatic change of the schematics. This change is because two compilations are now executed, one is saved in the `server` folder, and the other is saved in the `browser` folder. It's just a way of differences both compilations, if you don't like it I can reverse that change. 😄